docs/blog/newsletter-october-2017.rst

```diff
@@ -0,0 +1,81 @@
+* Print it out on paper and read/edit in the new format.
+* Read each sentence independently. Try reversing the order of sentences, starting with the last one and working your way backward to the first. Or reverse the order of paragraphs.
+* Listen to it being read aloud, either by you or by a device. Text-to-Speech tools are widely available.
+* For typos, repeats, and especially homophones, highlight a single line at a time. If it’s hardcopy, use a ruler. This keeps you from reading to the next line and helps you notice the “the the” problems.
+* Make the text much larger, or change the font. Folks recommended these tricks for on-screen or printed page reviews.
+* Last but not least, check out `this blog post of 10 tips for proofreading your own work <https://writetodone.com/get-your-eagle-eye-on-10-tips-for-proofreading-your-own-work/>`_. The post repeats some of our group's suggestions, and adds more.
+
+************************************************
+Defining developer relations/evangelism/advocacy
+************************************************
+This month, someone asked a question which many of us have wondered about, at one point or another: "What does a developer advocate or evangelist actually do?" (You may have even wondered this _as_ a devleoper advocate/evangelist...)
+
+The ensuing conversation made it clear that whether you call it developer relations (DevRel), developer evangelism, or developer advocacy, it can mean a lot of different things. But some common themes emerged:
+
+* Folks in these roles are generally expected to be the "face" of their developer community. Their work involves making their product/language/framework/etc. more visible and more approachable.
+* Often they sit at an overlap point between engineering – since they're (ideally) experts in the tool they're representing – and sales, marketing, documentation, etc.
+* The line between "evangelist" and "advocate" is blurry, but often the evangelist side is about bringing the product out to the community while the advocate side focuses on bring the community's voice back to the product team.
+* Although these roles may have gotten their start in the open source community, they're becoming increasingly common across the industry.
+
+For further reading on the topic, this medium post has lots of good thoghts: `https://medium.com/google-developers/the-core-competencies-of-developer-relations-f3e1c04c0f5b <https://medium.com/google-developers/the-core-competencies-of-developer-relations-f3e1c04c0f5b>`_
+
+*************************
+Upcoming community events
+*************************
+
+**WRITE THE DOCS DAY AUSTRALIA COUNTDOWN: Six Weeks Left!**
+We're so excited to be coming up on our first official event down under! If you're going to be in the Melbourne area in November, come a full day of sprints, talks, and community-building! Check it out (and get your ticket!) here: `http://www.writethedocs.org/conf/au/2017/ <http://www.writethedocs.org/conf/au/2017/>`_
```

Comment:
Should it be "...come for a full day..."?

Reply:
it certainly should